src/components/productDetail.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { useParams, Outlet } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getProductDetails } from '../redux/products/products';
+import './productDetail.css';
+
+const ProductDetail = () => {
+  const { productId } = useParams();
+  const product = useSelector((state) => state.selectedProduct);
+  const {
+    image, title, price, category, description,
+  } = product;
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (productId && productId !== '') dispatch(getProductDetails(productId));
+  }, [dispatch, productId]);
+
+  return (
+    <div className="ui grid container">
+      {Object.keys(product).length === 0 ? (
+        <div>...Loading</div>
+      ) : (
+        <div className="ui placeholder segment">
+          <div className="ui two column stackable center aligned grid">
+            <div className="ui vertical divider" />
+            <div className="middle aligned row">
+              <div className="column lp">
+                <img className="ui fluid image" src={image} alt={title} />
+              </div>
+              <div className="column rp">
+                <h1>{title}</h1>
+                <h2 className="ui teal tag label">
+                  $
+                  {price}
+                </h2>
+                <h3 className="ui brown block header">{category}</h3>
+                <p>{description}</p>
+                <div className="ui vertical animated button">
+                  <div className="hidden content">
+                    <i className="shop icon" />
+                  </div>
+                  <div className="visible content">Add To Cart</div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      )}
+      <Outlet />
+    </div>
+  );
+};
```

Comment:
- [x] The same thing here, kindly check my comment above ⬆
> Kindly, update the design for the details page to look similar to the original design 💯

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/206392516-b0b3194a-bf93-45ec-8c84-1daa66955ffc.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/205421212-9f967f45-c7ce-437f-aa22-7dc41c3c1468.PNG" width="500" /> |


- [ ] Please add a "Back" button that allows moving from the "Details" page to the "Home" page by clicking on it 👍

![image](https://user-images.githubusercontent.com/45231276/206393421-001577e4-a9e7-4945-94ed-61880de00596.png)

Reply:
@Bondok6 Hi there thanks for the honest review. I will most definitely work on the styling. However, because of the API that I chose, I think that it would be extremely difficult to match the details page with that of the template. I can most definitely work on inserting a back button but not really the detailed product design as it wouldn't make sense. The templates design makes sense because it works with fetching different country stats APIs, however since my API is a clothing shop it should mimic that of an ecommerce store. What would you recommend I do to get my design page still look similar to that of the template and still look like a professional ecommerce store. Also in my video, I did specifically point out two features that are my favourite and are interesting (the add to cart button and the search bar).